integration_tests/tests/web_app_certos.rs

```diff
@@ -465,6 +465,35 @@ mod website {
       check_public_certificate(&d, &title, &format!("Invitation {raw_description}"), &image).await;
     }
 
+
+    integration_test!{ use_graphiql (c, d)
+      async fn send_graphql_query(d : &Selenium, query: &str) {
+        d.goto("http://localhost:8000/#").await;
+        d.click("#graphiql").await;
+        d.driver.maximize_window().await.expect("to maximize window");
+        d.click(".graphiql-tab-add").await;
+        d.click(".graphiql-query-editor").await;
+        d.fill_in(".graphiql-query-editor textarea", query).await;
+        d.wait_for("#header-setted").await;
```

Comment:
Esto cambia.

Reply:
the `d.wait_for("#header-setted").await;` is modified by the following: 

`d.wait_for("#header-set").await;`